homeassistant/components/overkiz/alarm_control_panel.py

```diff
@@ -0,0 +1,306 @@
+"""Support for Overkiz alarm control panel."""
+from __future__ import annotations
+
+from collections.abc import Awaitable, Callable
+from dataclasses import dataclass
+from typing import cast
+
+from pyoverkiz.enums import OverkizCommand, OverkizCommandParam, OverkizState
+from pyoverkiz.enums.ui import UIWidget
+from pyoverkiz.types import StateType as OverkizStateType
+
+from homeassistant.components.alarm_control_panel import (
+    AlarmControlPanelEntity,
+    AlarmControlPanelEntityDescription,
+)
+from homeassistant.components.alarm_control_panel.const import (
+    SUPPORT_ALARM_ARM_AWAY,
+    SUPPORT_ALARM_ARM_HOME,
+    SUPPORT_ALARM_ARM_NIGHT,
+    SUPPORT_ALARM_TRIGGER,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import (
+    STATE_ALARM_ARMED_AWAY,
+    STATE_ALARM_ARMED_HOME,
+    STATE_ALARM_ARMED_NIGHT,
+    STATE_ALARM_DISARMED,
+    STATE_ALARM_PENDING,
+    STATE_ALARM_TRIGGERED,
+    Platform,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import HomeAssistantOverkizData
+from .const import DOMAIN
+from .entity import OverkizDescriptiveEntity
+
+
+@dataclass
+class OverkizAlarmDescriptionMixin:
+    """Define an entity description mixin for switch entities."""
+
+    supported_features: int
+    fn_state: Callable[[Callable[[str], OverkizStateType]], str]
```

Comment:
```
homeassistant/components/overkiz/alarm_control_panel.py:172: error: Argument "fn_state" to "OverkizAlarmDescription" has incompatible type:
"Callable[[Callable[..., Awaitable[Union[str, int, float, bool, Dict[Any, Any], List[Any], None]]]], str]"; 
expected "Callable[[Callable[[str], Union[str, int, float, bool, Dict[Any, Any], List[Any], None]]], str]"  [arg-type]
```

Not sure what is going wrong here. I implemented this the same way as the others. Anyone that has an idea?